resources/views/site/pilots/landing.blade.php

```diff
@@ -98,7 +98,7 @@
                 <div class="panel-body">
                     <ol>
                         <li>
-                            <a href="{{ route('visiting.landing') }}" rel="noreferrer noopener">Apply to visit as a Pilot</a>
+                            <a href="{{ route('visiting.landing') }}" rel="noreferrer noopener">Apply to visit as a Pilot</a> not seeing the option you want? Submit a ticket <a href="https://helpdesk.vatsim.uk/open.php" target="_blank" rel="noreferrer noopener">here</a>
```

Comment:
This potentially needs a comma or something to break up the sentence - otherwise looks great!